docker-notes

```diff
@@ -0,0 +1,21 @@
+Steps to reproduce GitHub Actions locally, in a docker image
+
+Locally:
+
+docker run --rm -it ubuntu
+
+Inside the image:
+
+apt-get update
+apt install curl git vim-nox less
+apt-get install libgtk2.0-0 libgtk-3-0 libgbm-dev libnotify-dev libgconf-2-4 libnss3 libxss1 libasound2 libxtst6 xauth xvfb
```

Comment:
any reason not to just have a Dockerfile?

Reply:
I would have to write and test it...

Reply:
I mean, no. :)